openpower/package/pkg-versions.mk

```diff
@@ -0,0 +1,137 @@
+################################################################################
+#
+# pkg-versions
+#
+# Read VERSION.readme in the current directory to learn about the version
+# string structure
+#
+################################################################################
+
+define OPENPOWER_SUBPACKAGE_VERSION
+
+$(2)_VERSION_FILE = $$(OPENPOWER_VERSION_DIR)/$(1).version.txt
+ALL_SUBPACKAGE_VERSIONS += $$($(2)_VERSION_FILE)
+
+### Create subpackage patch file
+define $(2)_OPENPOWER_PATCH_FILE
+
+mkdir -p "$$(OPENPOWER_VERSION_DIR)"
+
+cd "$$(PKGDIR)"; if ls *.patch; then sha512sum *.patch | sha512sum | cut -c 1-7 | \
```

Comment:
Yeah there could be something wrong with the external.mk change, I'm trying things now. I originally did the cd to just have an example of what the variable looks like.
I couldn't get ls *.patch $path/$(1) to work without cd'ing into the dir first, so I'm trying find now
$$(foreach path, $$(BR2_GLOBAL_PATCH_DIR),find $$(path)/$(1) -name "*.patch" -printf '%f\n')